phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -2239,8 +2238,7 @@ public void dropTable(RpcController controller, DropTableRequest request,
 
             if (pTableType == PTableType.TABLE || pTableType == PTableType.VIEW) {
                 // check to see if the table has any child views
-                try (Table hTable = ServerUtil.getHTableForCoprocessorScan(env,
-                        getSystemTableForChildLinks(clientVersion, env.getConfiguration()))) {
+                try (Table hTable = ServerUtil.getHTableSystemChildLink(env)) {
```

Comment:
nit: Rename to `getHTableSysChildLinkOrSysCat` or something to indicate that we may get the latter too